backend/open_webui/models/groups.py

```
@@ -189,5 +185,24 @@ def delete_all_groups(self) -> bool:
+
+                for group in groups:
+                    group.user_ids.remove(user_id)
+                    db.query(Group).filter_by(id=group.id).update(
+                        {
+                            "user_ids": group.user_ids,
+                            "updated_at": int(time.time()),
+                        }
+                    )
+                    db.commit()
+
+                return True
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)